ee/vellum_ee/workflows/display/tests/workflow_serialization/test_basic_map_node_serialization.py

```diff
@@ -392,3 +392,50 @@ def test_serialize_workflow():
             "workflow",
         ],
     }
+
+
+def test_serialize_workflow__map_node_output_value_structure():
+    """
+    Test that map node subworkflow outputs have correct value structure instead of None.
+    """
+    # GIVEN a Workflow that uses a MapNode with outputs
+    workflow_display = get_workflow_display(workflow_class=SimpleMapExample)
+
+    # WHEN we serialize it
+    serialized_workflow: dict = workflow_display.serialize()
+
+    # THEN the map node's subworkflow should have outputs with proper value references
+    workflow_raw_data = serialized_workflow["workflow_raw_data"]
+    map_nodes = [node for node in workflow_raw_data["nodes"] if node["type"] == "MAP"]
+
+    assert len(map_nodes) == 1
+    map_node = map_nodes[0]
+
+    # AND the map node should have subworkflow data
+    assert "data" in map_node
+    assert "workflow_raw_data" in map_node["data"]
+    subworkflow_data = map_node["data"]["workflow_raw_data"]
+
+    # AND the subworkflow should have nodes with outputs
+    nodes_with_outputs = [node for node in subworkflow_data["nodes"] if "outputs" in node]
+    assert len(nodes_with_outputs) == 1
```

Comment:
We're don't want to check the subworkflow with this test. We want a Workflow that has a final output node that references the map node, then we check the serialized data of that final output node.

`SimpleMapExample` doesn't have this structure so we'll need to generate a new workflow inline in order to test this. Therefore, we like should move this test out of this file